packages/components/src/components/componentTypes.spec.ts

```diff
@@ -15,6 +15,9 @@ describe.each<TagName>(TAG_NAMES.filter((x) => !INTERNAL_TAG_NAMES.includes(x)))
   const sourceFileContent = fs.readFileSync(sourceFilePath, 'utf8');
 
   const { props = {}, eventNames = [] } = getComponentMeta(tagName);
+  if (tagName === 'p-multi-select') {
```

Comment:
Left over or still relevant? Would be unfortunate if we would now need a condition here.